src/EnergyPlus/UnitarySystem.cc

```diff
@@ -17891,6 +17964,13 @@ namespace UnitarySystems {
                                              state.dataUnitarySystems->unitarySys[sysNum].m_EMSOverrideCoilSpeedNumOn,
                                              state.dataUnitarySystems->unitarySys[sysNum].m_EMSOverrideCoilSpeedNumValue);
                         }
+                        SetupEMSActuator(state,
+                                         "Coil Speed Control",
```

Comment:
Is it fine using the same actuator name as main multispeed heating/cooling coils?